integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/receive/InternalQueue.java

```diff
@@ -89,8 +90,11 @@
     public void noarkListener(byte[] message, Session session) {
         MDC.put(MoveLogMarkers.KEY_ORGANISATION_NUMBER, properties.getOrg().getNumber());
         EduDocument eduDocument = documentConverter.unmarshallFrom(message);
-
-        forwardToNoark(eduDocument);
+        try {
+            forwardToNoark(eduDocument);
+        } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)